File: Server/Toopeto/JsonPacket/UserPacket.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;
using System.Text;
using AnarSoft.Utility.Utilities;

namespace Toopeto.JsonPacket
{
    public class UserPacket : Json
    {
        [DataMember]
        public string userName;

        [DataMember]
        public string Signature;
        [DataMember]
        public string avatar;

        public UserPacket():base()
        {
            this.packetName = PacketNameEnum.user.ToString();
          
        }
        public override string getString()
        {
            return JsonSerializer.Serialize(this);
        }
        public static UserPacket GetUserPacket(string JsonPacket)
        {
            return JsonSerializer.DeSerialize<UserPacket>(JsonPacket);
        }

    }
}

File: ClientAndroid/src/main/java/ir/bilgisoft/toopeto/json/UserPacket.java
package ir.bilgisoft.toopeto.json;

import org.json.JSONException;

import com.google.gson.Gson;

import java.util.UUID;

import ir.bilgisoft.toopeto.entities.Account;

/**
 * Created by mehrang on 03/03/2015.
 */
public class UserPacket extends Json {

    public String userName;

    public String password;

    public String Signature;

    public UserPacket()
    {
        super();
        this.packetName =Enums.PacketNameEnum.user.toString();

    }

    @Override
    public  String getString()
    {
        Gson gson = new Gson();
        String json = gson.toJson(this);
        return json;
    }

    public static UserPacket GetUserPacket(String JsonPacket)
    {
        Gson gson = new Gson();
        UserPacket up= gson.fromJson(JsonPacket,UserPacket.class);
        return  up;
    }



}
